csharp_dotnetcore/17.Proactive-Messages/README.md

```diff
@@ -0,0 +1,41 @@
+This sample demonstrates how to send proactive messages to users by
+capturing a conversation reference, then using it later to initialize
+outbound messages using ASP.Net Core 2.
+
+# Concepts introduced in this sample
+**Proactive messaging**. Often bots send _reactive messages_, a message based on the user's last input. However, we might need to send a proactive message, one not based on the user's input.
+
+For example, when your bot needs to perform a task that can take an indeterminate amount of time, you can store information about the task, tell the user that the bot will get back to them when the task finishes, and let the conversation proceed. When the task completes, the bot can send the confirmation message proactively on the same conversation.
+
+
+
+# To try this sample
+
+- Clone the samples repository
+```bash
+git clone https://github.com/Microsoft/botbuilder-samples.git
+```
+
+
+## Visual studio
+- Navigate to the samples folder (`BotBuilder-Samples\csharp_dotnetcore\17.Proactive-Messages`) and open `ProactiveBot.csproj` in Visual studio 
```

Comment:
Casing. Both here and in the line below. ("Visual Studio")

Reply:
Fixed.